utils/GenerateSpritesheet.java

```diff
@@ -5,40 +5,43 @@
  * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with FreeRCT. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import java.awt.*;
-import java.awt.event.*;
 import java.awt.image.*;
 import java.io.*;
-import java.util.*;
 import javax.imageio.ImageIO;
-import javax.swing.*;
 
 /**
  * This program provides an easy way to generate spritesheets for ride animations from individual (blender) images.
  * Let's assume you've made an animation with `f` frames for a ride with a size of `x` × `y` tiles.
  * The base images, all of which are of size `w` × `h`, are located in `src_dir`.
  * The images are called "PREFIX0000.png" through "PREFIX9999.png", where `PREFIX` may be different for each of the four rotations
- * (let's call the prefixes `se`,`ne`,`nw`,`sw` in this order). The number of the animation's starting frame is `idOff`.
+ * (let's call the prefixes `se`,`ne`,`nw`,`sw` in this order). The number of the
+ * animation's starting frame is `idOff_<p>` for each of the four prefixes <p>.
  * To generate the spritesheet and save it as `output_file`, run
- *    java GenerateSpritesheet src_dir output_file idOff se ne nw sw f x y w h
+ *    java GenerateSpritesheet src_dir output_file idOff_se se idOff_ne ne idOff_nw nw idOff_sw sw f x y w h true
+ * The last, optional parameter can be set to `false` to disable intelligent calculation of frame indices
+ * (useful when (ab)using this tool to generate spritesheets for a different purpose than the one stated above).
```

Comment:
You're entering dangerous territory here, in a year or two you won't remember exactly what this "intelligent calculation was, or if you do, you won't remember what the other behavior is.

To be fair, I never documented the sprite indexing in the rcd input files either, so I won't say you have to document it, but please be careful what you don't explain.